sopa/io/histopathology.py

```diff
@@ -99,7 +99,7 @@ def _default_image_models_kwargs(image_models_kwargs: dict | None) -> dict:
 def _open_wsi(path: str | Path) -> tuple[str, xarray.Dataset, Any, dict]:
     import tiffslide
 
-    image_name = Path(path).absolute().name.split(".")[0]
+    image_name = Path(path).stem
```

Comment:
Does it happen to have multiple extensions (like `.ome.tif`) or is it always one? Because `.stem` only removes the "last" extension, i.e. `image.ome.tif` would become `image.ome`

Reply:
hmm I did this because there are some filename that include `.` before the file extension. Here is an example from TCGA:  `TCGA-73-4658-01A-01-BS1.2859e392-66d8-407e-85db-25bf4e4c5ff3.svs` but there are more examples like this at the in-house databases with more segments in the filename that are separated with a `.`. Maybe we need a filetype detection function potentially the detect_format function of tiffslide can help?

Reply:
Maybe we could implement something like this, yes. Do you have a list of possible WSI extensions? Mostly, which one has a "double" extension?

But maybe we can keep your current implementation for now and open a new PR later if needed?